data/seeds/003_businesses.js

```diff
@@ -0,0 +1,32 @@
+exports.seed = function (knex) {
+  // Deletes ALL existing entries
+  return knex('businesses')
+    .del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('businesses').insert([
+        {
+          user_id: '00ulthapbErVUwVJy4x6',
+          email: 'groomerexpress@yahoo.com',
+          password: 'groomer',
+          name: 'Groomer Express',
+          profile_photo:
+            'https://imageproxy.themaven.net//https%3A%2F%2Fedm.com%2F.image%2FMTc0NjYyNjI0ODg4MjM1Mzg2%2Fscience-of-earworms-explain-why-never-gonna-give-you-up-is-stuck-in-our-heads-30-years-later.png',
```

Comment:
Don't mine me being nitpicky here: I think we should shorten long urls so the returned data on the client side looks cleaner. We could use [Bitly](https://bitly.com/) for this 😄 
```suggestion
            'https://bit.ly/2TWHF9n',
```

Reply:
That's a great idea! I will change that and use bitly in the future.